mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -108,6 +108,8 @@
     <string name="mozac_feature_addons_updater_notification_title">%1$s has a new update</string>
     <!-- The content of the notification, this will be shown when an add-on needs new permissions, to be updated. %1$d is the amount of new permissions-->
     <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
+    <!-- The content of the notification, this will be shown when an add-on needs a new permission-->
+    <string name="mozac_feature_addons_updater_notification_content_singular">A new permissions is required</string>
```

Comment:
@Amejia481 
English is broken here: "a permissions is required".

Less important, I'd really like to improve all these comments: "The content of the notification displayed/shown when …". I hat to re-read it a couple of times to make sense of the sentence.

Reply:
👍